duckduckgo/Android
# Update app id in preparation for release

Asana Issue URL: https://app.asana.com/0/488551667048375/524078036165561

## Description
Updated the app id and removes no longer needed db migration data


## Steps to Test this PR:
1. Install the app
1. Check carefully that the app id matches the one currently in the play store (see image below)
1. Ensure that future db migrations still work (as old db data was deleted)
<img width="890" alt="screen shot 2018-01-15 at 17 44 37" src="https://user-images.githubusercontent.com/1652407/34955327-caa2a5a2-fa1b-11e7-99cd-22d3921339b8.png">
